client/src/app/app.module.css

```diff
@@ -0,0 +1,43 @@
+button {
+	line-height: 100%;
+	text-align: center;
+	border: 0.3vh solid #000;
+	text-shadow: 0.2vh 0.2vh #000;
+	box-shadow: inset -0.3vh -0.5vh #0006, inset 0.3vh 0.3vh #fff7;
+}
+button:hover {
+	text-shadow: 0.3vh 0.3vh #202013cc;
+}
+
+input {
+	border: 0.3vh solid #000;
+}
+input:focus-visible {
+	outline-offset: 0;
+	outline: 0.5vh solid #333;
+	border-radius: 0.5vh;
+}
+
+::-webkit-scrollbar {
```

Comment:
These will not fully work in FF. It is okay to have not have it FF but we should check we it loooks okay-ish.

Reply:
I've tested it on firefox, can confirm that the scrollbar stays as the default on firefox. However, it doesn't cause any issues, and the default scrollbar actually looks quite nice.